package.json

```diff
@@ -42,13 +42,12 @@
     "vscode-languageserver-textdocument": "^1.0.0"
   },
   "devDependencies": {
-    "@types/sinon": "^10.0.0",
     "@types/tape": "^4.0.0",
     "c8": "^7.0.0",
+    "execa": "^6.0.0",
```

Comment:
this was added, but doesn't appear to be used?

Reply:
Did you expand `test/index.js`?

Reply:
I did not, got it